bd/tryLogin.go

```diff
@@ -0,0 +1,20 @@
+package bd
+
+import (
+	"github.com/SaiKumarMalve/Gator-News/models"
+	"golang.org/x/crypto/bcrypt"
+)
+
```

Comment:
Write a brief comment of what function does for maintainability purpose

Reply:
I will write the comments.